### ISSUE TITLE
Flail mitral valve leaflet

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10431]
**Preferred term label:**
Flail mitral valve leaflet

**Synonyms**
papillary muscle rupture

**Definition (free text, please give PubMed ID)**
failure of leaflets drawing together due to ruptured chordae tendineae or papillary muscle 
https://uts.nlm.nih.gov/uts/umls/concept/C0746594
https://uts.nlm.nih.gov/uts/umls/concept/C3670889
https://uts.nlm.nih.gov/uts/umls/concept/C2023033
https://uts.nlm.nih.gov/uts/umls/concept/C2023034

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Papillary muscle rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think the following term is essentially a synonym?
There is a spelling error or variant (an "e" is missing)
[Mitral chordae tendinae rupture HP:0034419](https://hpo.jax.org/app/browse/term/HP:0034419)

**Comment by MickeySegal** [comment]
Both spellings are in use: https://www.google.com/search?q=chordae+tendineae
I don't know which is correct.
https://hpo.jax.org/app/browse/term/HP:0034419 seems fine, and it may be worth adding flail mitral valve to the entry.

**Comment by pnrobinson** [comment]
@MickeySegal 
added synonym.
